resources/views/tasks/assign.blade.php

```diff
@@ -0,0 +1,23 @@
+@extends('layouts.app')
+
+@section('content')
+
+
+<form action="/tasks/{{ $task->id}}/assign" method="post">
+    @csrf
+    <div class="form-group">
+        <label for="assign_to_user_id">Assign task {{ $task->name }} to</label>
+        <select class="form-control" id="assign_to_user_id" name="assign_to_user_id">
+            <option>Select user</option>
```

Comment:
Find all <option></option> that don't have a default value and set it to value="". This way your backend validation will pick it up and show the relevant error message.

Reply:
done